tests/system/Config/fixtures/.env

```diff
@@ -6,4 +6,5 @@ NULL=
 
 SimpleConfig.onedeep=baz
 SimpleConfig.default.name=ci4
-simple.name=simpleton
\ No newline at end of file
+simple.name=simpleton
+impleconfig.shortprefix=wht
```

Comment:
Haha~ When I'm doing it, I just think it is wired. You can add it into issue. If I find more bugs I will tell you. :)